apps/glusterfs/volume_durability_test.go

```diff
@@ -77,41 +77,41 @@ func TestNoneDurability(t *testing.T) {
 	r := &NoneDurability{}
 	r.SetDurability()
 
-	gen := r.BrickSizeGenerator(100 * GB)
+	gen := r.BrickSizeGenerator(50 * GB)
```

Comment:
Instead of reducing the initial size, from 100 to 50 here, why not just ad another block below? reduced the diff and makes the change more obvious?